### ISSUE TITLE
[Question] attention_mask for shorter inputs

### ISSUE THREAD

**Issue by see--** [CyberZHG/keras-bert#67]
**Is your feature request related to a problem? Please describe.**
First of all, thanks for this great repo! I used your colab classification example as reference. Where can I put the `attention_mask` for sentences that are shorter than max_len? In PyTorch it is described as follows: https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/README.md

> attention_mask: an optional torch.LongTensor of shape [batch_size, sequence_length] with indices selected in [0, 1]. It's a mask to be used if some input sequence lengths are smaller than the max input sequence length of the current batch. It's the mask that we typically use for attention when a batch has varying length sentences.

**Describe the solution you'd like**
The model input should be a list with len 3: [indices, np.zeros_like(indices), attenion_mask]

**Describe alternatives you've considered**
The model already has 3 inputs. I think the 3rd one is used for the pretraining mask.

**Comment by CyberZHG** [comment]
`attention_mask` is generated from the embedding layer (see `mask_zeros`)

**Comment by see--** [comment]
Ok, thanks. To sum it up: This repo takes care of masking zeros, but the results will be different from the PyTorch version: https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/pytorch_pretrained_bert/modeling.py#L712-L741.

**Comment by CyberZHG** [comment]
Actually, the results are approximately the same. :doge:

**Comment by see--** [comment]
Shouldn't both outputs be closer, if the masking is working?
```
import os
import tensorflow as tf
import numpy as np
os.environ['TF_KERAS'] = '1'  # noqa
import codecs
from keras_bert import Tokenizer, load_trained_model_from_checkpoint


cache_subdir = os.path.expanduser('~/bert_data/uncased_L-12_H-768_A-12')
vocab_path = os.path.join(cache_subdir, 'vocab.txt')

model = load_trained_model_from_checkpoint(
    os.path.join(cache_subdir, 'bert_config.json'),
    os.path.join(cache_subdir, 'bert_model.ckpt'),
    training=True,
    trainable=False,
    seq_len=128,
)
inputs = model.inputs[:2]
x = model.get_layer('NSP-Dense').output
x = tf.keras.layers.Dense(1, activation='sigmoid')(x)
model = tf.keras.models.Model(inputs, x)

token_dict = {}
with codecs.open(vocab_path, 'r', 'utf8') as reader:
  for line in reader:
    token = line.strip()
    token_dict[token] = len(token_dict)
tokenizer = Tokenizer(token_dict)

ids, _ = tokenizer.encode("Some text unaffable.", max_len=128)
ids = np.float32(ids)
ids_rolled = np.roll(ids, 30)
ids = np.float32([ids, ids_rolled])

out_keras = model.predict([ids, np.zeros_like(ids)])
print(out_keras)
```

Output
```
array([[0.28907815],
       [0.29562896]], dtype=float32)
```
